postgresql-custom-query.yml.sample

```diff
@@ -56,9 +56,12 @@ queries:
     sample_name: PostgresMissingIndexesSample
 
   # Query to collect most expensive queries. This query needs to repeat for every user database to collect data from all of them.
-  # Note this extension may not be enabled on your server. 
+  # Note this extension may not be enabled on your server.
+  # In the main postgres-config.yml file it is best to let it default to the postgre database when connecting. 
```

Comment:
```suggestion
  # In the main postgres-config.yml file it is best to let it default to the postgres database when connecting. 
```

Reply:
Thanks for spying my typo. It's fixed now.